src/client/app/components/RouteComponent.tsx

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import * as React from 'react';
-import { RedirectFunction, Route, Router, RouterState } from 'react-router';
+import { Route, Router, Switch, Redirect } from 'react-router-dom';
```

Comment:
I just want to note that the changes seem to work but I'm not up on the auth code.